README.md

```diff
@@ -39,69 +43,131 @@ Pre-built docker for popular AI frameworks. Easy to include heterogeneous hardwa
 
-## How to use
-### How to train jobs
-- How to write OpenPAI jobs
-    - [Quick start: how to write and submit a CIFAR-10 job](./examples/README.md#quickstart)
-    - [Write job from scratch in depth](./docs/job_tutorial.md)
-    - [Learn more example jobs](./examples/#offtheshelf)
-- How to submit OpenPAI jobs
-    - [Submit a job in Web Portal](./docs/submit_from_webportal.md)
-    - [Submit a job in Visual Studio](https://github.com/Microsoft/vs-tools-for-ai/blob/master/docs/pai.md)
-    - [OpenPAI VS Code Extension](./contrib/pai_vscode)
-- How to run AutoML jobs on OpenPAI
-    - [Submit a job in Neural Network Intelligence](https://github.com/Microsoft/nni/blob/master/docs/PAIMode.md)
-- How to request on-demand resource for in place training
-    - [Launch a jupyter notebook and work in it](./examples/jupyter/README.md)
+### Train first model
 
-### Cluster administration
-- [Deployment infrastructure](./docs/pai-management/doc/distributed-deploy.md)
-- [Cluster maintenance](https://github.com/Microsoft/pai/wiki/Maintenance-(Service-&-Machine))
-- [Monitoring](./docs/webportal/README.md)
+Follow [here](examples/README.md#quickstart) to create your first job definition and have a look on job definition. Then refer to [here](docs/submit_from_webportal.md) to submit it to OpenPAI. It may be a very simple job, as it downloads data and code from internet, and doesn't copy model back to you. It's used to let you understand OpenPAI job definition and familiar with Web portal.
 
-## Resources
+### Learn deeper on job definition
 
-- The OpenPAI user [documentation](./docs/documentation.md) provides in-depth instructions for using OpenPAI
-- Visit the [release notes](https://github.com/Microsoft/pai/releases) to read about the new features, or download the release today.
-- [FAQ](./docs/faq.md)
+* Choose training environment. OpenPAI uses [Docker](https://www.docker.com/) to provide runtime environment.
+
+    You can refer to [here](https://hub.docker.com/r/ufoym/deepo) to find more deep learning environments, for example, ufoym/deepo:pytorch-py36-cu90. Note, it doesn't include openssh-server, curl. So you want to use ssh with those docker images, you need use `apt install openssh-server curl` in command field to enable ssh.
+
+* Put code and data in. OpenPAI creates a clean environment, you need to use co mmand field to copy data and code in, and start training. You can join multiple commands with `&&`. If extra system or Python components are needed, they can be installed in the command by `apt install` or `python -m pip install`.
+
+    There are some suggestions to exchange data with running environment, but you should check with OpenPAI administrator, which kind of storage is supported, and recommended approach to access it. Refer to [here](TODO provide storage solution) for sample solutions.
```

Comment:
> is supported

is supported by the OpenPAI instance/cluster.

>recommended approach

recommend approach

Reply:
update for #1. For #2, recommended is adj., and recommend is verb only. so keep recommended here.